### ISSUE TITLE
Packages for annotations

### ISSUE THREAD

**Issue by TomSmithCGAT** [bioinformatics-core-shared-training/Bulk_RNAseq_Course_Base#11]
'Annotation and Visualisation of Differential Expression Results' explains there are multiple ways to obtain annotations and then demonstrates how to do this with `AnnotationHub`, which seems perfectly reasonable.

https://github.com/bioinformatics-core-shared-training/Bulk_RNAseq_Course_Base/blob/1169a4f2536ff83b69d0f748ca6d700b3406ce23/Markdowns/11_Annotation_and_Visualisation.Rmd#L57-L59

However, after this, we then load a set of annotations that was obtained via `biomaRt`. I assume this is a legacy issue. This is a bit confusing and I would suggest the loaded annotations should be obtained with `AnnotationHub` too.
https://github.com/bioinformatics-core-shared-training/Bulk_RNAseq_Course_Base/blob/1169a4f2536ff83b69d0f748ca6d700b3406ce23/Markdowns/11_Annotation_and_Visualisation.Rmd#L162-L182

**Comment by AshKernow** [comment]
Yes, we should look at resolving this discrepancy. We no longer need the Chr, Start etc columns, just Symbol and Entrez.
